config/jobs/kubernetes/sig-node/dra.jinja

```diff
@@ -0,0 +1,120 @@
+    skip_report: false
+    {%- endif %}
+    {%- endif %}
+    labels:
+      preset-service-account: "true"
+      {%- if use_dind == "true" %}
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      {%- endif %}
+      {%- if job_type == "node" %}
+      preset-k8s-ssh: "true"
+      {%- endif %}
+      {%- if job_type == "node" %}
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
+      {%- endif %}
+    annotations:
+      testgrid-dashboards: {{testgrid_dashboards}}
+      description: {{description}}
+      testgrid-alert-email: {{testgrid_alert_email}}
+      {%- if kind != "canary" %}
+      fork-per-release: "true"
+      {%- endif %}
+    decorate: true
+    decoration_config:
+      timeout: {{timeout}}
+    path_alias: k8s.io/kubernetes
+    {%- if job_type == "node" %}
+    extra_refs:
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    {%- endif %}
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20241218-d4b51bc3e8-master
+        command:
+        - runner.sh
+        {%- if job_type == "node" %}
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --deployment=node
+        {%- if inject_ssh_public_key == "true" %}
+        - --env=KUBE_SSH_USER=core
+        {%- endif %}
+        - --gcp-zone=us-west1-b
+        - '--node-test-args=--feature-gates=DynamicResourceAllocation=true --service-feature-gates=DynamicResourceAllocation=true --runtime-config=api/beta=true --container-runtime-endpoint=unix:///{%-if runtime == "crio" %}var/{%- endif %}run/{{runtime}}/{{runtime}}.sock --container-runtime-process-name=/usr/{%- if runtime == "crio" %}local/{%- endif %}bin/{{runtime}} --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/{{runtime}}.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\"name\": \"{{runtime}}.log\", \"journalctl\": [\"-u\", \"{{runtime}}\"]}"'
+        - --node-tests=true
+        - --provider=gce
+        - '--test_args=--timeout=1h --label-filter="{{label_filter}}"'
+        - --timeout=65m
+        - --node-args=--image-config-file={{image_config_file}}
+        {%- if inject_ssh_public_key == "true" %}
+        env:
+        - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
+          value: "1"
+        - name: GOPATH
+          value: /go
+        {%- endif %}
+        {%- else %}
+        args:
+        - /bin/sh
+        - -xce
+        - |
+          make WHAT="github.com/onsi/ginkgo/v2/ginkgo k8s.io/kubernetes/test/e2e/e2e.test"
+          curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" kind
+          kind build node-image --image=dra/node:latest .
+          trap 'kind export logs "${ARTIFACTS}/kind"; kind delete cluster' EXIT
+          {%- if kind == "canary" %}
```

Comment:
Why does this check "canary"?

This seems to overload the meaning of "canary":
- For canary PR jobs (= dra-canary.yaml).
- The "all features enabled" CI and presubmit jobs.

Let's use "canary" for "part of dra-canary.yaml" and something else for feature gates. How about "alpha"?

Reply:
`with_all_features` was set to `true` only for two canary jobs in your PR.
However, I agree that a separate option would be more appropriate here. How about `all_features`?

Reply:
Works for me.